### ISSUE TITLE
Symbols for male and female are hard to read!

### ISSUE THREAD

**Issue by mhkc** [Clinical-Genomics/scout#2697]
The symbols small and hard to read. Since the sex of a patient is a important factor this could in lead to erroneous conclusions.

![image](https://user-images.githubusercontent.com/12696913/120978182-7f408280-c774-11eb-885d-a5b8fbd597b4.png)

I think we should increase the figure size and possibly change the figure to a more clear pictogram. Alternatively use M/F to designate male and female respectively.

**Comment by northwestwitch** [comment]
Agreed. We could check perhaps in [fontawesome 5](https://fontawesome.com/v5.15/icons?d=gallery&p=2) if there are icons that are more clear to understand!

**Comment by dnil** [comment]
I guess perhaps mars/venus is clearer. But adding back M/F is a very safe bet.

**Comment by northwestwitch** [comment]
@moedarrah an issue for your perhaps?
